src/fabric/src/fabric2_db.erl

```diff
@@ -794,9 +801,9 @@ fold_local_docs(Db, UserFun, UserAcc0, Options) ->
 
             UserAcc1 = maybe_stop(UserFun({meta, Meta}, UserAcc0)),
 
-            UserAcc2 = fabric2_fdb:fold_range(TxDb, Prefix, fun({K, V}, Acc) ->
+            UserAcc2 = fabric2_fdb:fold_range(TxDb, Prefix, fun({K, _V}, Acc) ->
```

Comment:
Oh, on second glance I think you should add a separate function like "get_local_doc_rev" that takes the value and returns the rev as that'll save you a full round trip here since we're only wanting the rev anyway.

Reply:
Made a `get_local_doc_rev/3` in `fabric2_fdb`.

https://github.com/apache/couchdb/pull/2267/files#diff-32274bcb4785f432811085d2e08c3580R531

Added extra belts and suspenders around that since both old and new values are binaries. Is that too much?

Reply:
Looks mostly fine. I might tried doing it in a case instead of the function head match but I could see that being not much prettier.